apps/mull-ui/src/app/components/modal/mull-modal/mull-modal.spec.tsx

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
 import { render } from '@testing-library/react';
-
+import React from 'react';
 import MullModal from './mull-modal';
 
 describe('MullModal', () => {
   it('should render successfully', () => {
-    const { baseElement } = render(<MullModal />);
+    const { baseElement } = render(
+      <MullModal
+        open={true}
+        setOpen={() => {
+          /* noop */
+        }}
```

Comment:
I'm a little confused if this is part of the ticket

Reply:
Its not, its to fix console errors which pop up during tests